assets/src/scss/layout/_navbar.scss

```diff
@@ -264,67 +220,96 @@ a.nav-link {
   }
 }
 
-.active a.nav-link {
-  --nav-link-active-- {
-    color: var(--nav-link--color, $white);
-    text-decoration-color: $gp-green;
-  }
-
-  opacity: 1;
-  text-decoration-line: underline;
-  text-decoration-thickness: 3px;
-  text-underline-offset: auto;
-  text-underline-position: under;
+// Underline animation with border-bottom
+// Using > * because there are some cases where
+// there a container, wrapped by the nav-item
+// is wrapping the nav-item
+a.nav-link:hover:before,
+.nav-item:not(.burger-menu-item):hover > a.nav-link:before,
+.active > a.nav-link:before,
+.active > * > a.nav-link:before {
+  transform: scaleX(1);
+  transition: transform 0.25s;
+  z-index: 1;
 
   @include large-and-up {
-    text-decoration-line: none;
-    text-decoration-thickness: 4px;
+    border-bottom-width: 4px;
   }
 }
 
-// Underline animation with border-bottom
-a.nav-link:hover:before,
-.active a.nav-link:before {
+.nav-donate {
+  padding: $sp-2 0;
+
   @include large-and-up {
-    transform: scaleX(1);
-    transition: transform 0.25s;
+    padding: 0;
   }
 }
 
-.nav-menu-toggle {
-  background: transparent;
-  border: none;
-  border-radius: 4px;
-  padding: 0 24px;
+.nav-donate:hover .dropdown::after {
+  transform: rotate(-90deg);
+}
+
+.nav-item,
+.nav-donate {
+  color: inherit;
+  font-size: $font-size-sm;
+  font-weight: 700;
 
   @include large-and-up {
-    display: none;
+    line-height: $menu-height;
+
+    &:hover {
+      .nav-submenu {
+        display: flex;
+        z-index: 999;
+      }
+    }
   }
 }
 
-.nav-menu-close {
-  background-color: var(--top-navigation--color, $white);
-  border: none;
-  display: block;
-  height: 68px;
-  inset-inline-end: 0;
-  mask: url("../../images/cross.svg") 50% 50%/16px 16px no-repeat;
-  position: absolute;
-  top: 0;
-  width: 68px;
-  z-index: 5;
+.btn-donate {
```

Comment:
This actually needs to be inside of `.top-navigation`. We also have [CSS targeting the clsas outside the nav](https://github.com/greenpeace/planet4-master-theme/blob/load-query-loop-from-html/assets/src/scss/components/_buttons.scss#L176-L201).